helm/cosmo/charts/router/templates/gateway.yaml

```diff
@@ -0,0 +1,64 @@
+{{- if .Values.istioGateway -}}
```

Comment:
Does the config requires a specific Istio version? If yes, please make a comment in the config.